config/migrations/2023/20231112161533-update-phone-numbers.sparql

```diff
@@ -0,0 +1,86 @@
+DELETE {
+  GRAPH <http://mu.semte.ch/graphs/administrative-unit> {
+  ?s <http://schema.org/telephone> ?o .
+  }
+}
+INSERT {
+ GRAPH <http://mu.semte.ch/graphs/administrative-unit> {
+   ?s <http://schema.org/telephone> ?newPhoneNumber .
+ }
+}
+WHERE {
+  GRAPH <http://mu.semte.ch/graphs/administrative-unit> {
+    ?s <http://schema.org/telephone> ?o .
+  }
+  
+  FILTER (SUBSTR(?o, 1, 1) != "+" AND STRLEN(STR(?o)) > 4 and !regex(?o, ".*[a-zA-Z].*", "i"))
```

Comment:
Minor remark, you can use `STRSTARTS(?o, "+")` instead of `SUBSTR(?o, 1, 1) != "+"`.